### ISSUE TITLE
Cannot add settings for properties

### ISSUE THREAD

**Issue by evless** [qlik-oss/nebula.js#500]
## 🐛 Bug report

Hi! I don't know, maybe it's not a bag. But if I use properties I will not see settings for change them. When I use it local then I see fields for change properties, but if I build component and use it in qlik I will not see fields with settings. Maybe I need to do something else? Or if I create component with nebula I cannot change properties into qlik?

Sorry that I created bug report, but I cannot find information about it :(

### Versions

- nebula.js: 1.0.0

**Comment by rzenere-synthese** [comment]
I have the same issue.
I tried to modify the sn-mekko-chart v1.0.4 (in order to change some of the displaying options) but after a long trying I wasn't able to see the Colors and Legends menu.
Like you, I've seen that this issue didn't happen with previous versions of nebula.js (I would point out to _**stardust**_, but I'm not totally sure) as in version 1.0.0 (which used an older release of stardust) everything worked smoothly

In the end I 'solved', after a long trying, doing a partial build with the following command: **_nebula sense --meta src/meta.json --partial_**
To use it in QS Enterprise, I copied the generated files (all _dist_ folder and the .js/.qext files that have been generated in the root folder) in a folder with the name of the extension and zipped it for QS Enterprise.
For QS Desktop you would have to copy the files in a folder under C:\Users\<your user>\Documents\Qlik\Sense\Extensions.

I hope this could help,
Riccardo
